lib/absinthe/type/custom.ex

```diff
@@ -2,7 +2,13 @@ defmodule Absinthe.Type.Custom do
   use Absinthe.Schema.Notation
 
   @moduledoc """
-  This module contains additional data types.
+  This module contains the following additional data types:
+  - datetime
```

Comment:
Of course, done. :) I hope you don't mind the force push. This way the history looks cleaner.